### PR TITLE
Added ElectricResistivity, Illuminance, LuminousFlux, LuminousIntensity, MagneticField.

### PR DESCRIPTION
**Pull request by baxcoders**
ElectricResistivity - OhmMeter (Ω·m)
Illuminance - Lux (lx)
LuminousFlux - Lumen (lm)
LuminousIntensity - Candela (cd)
MagneticField - Tesla (T)

### CODE REVIEW

File: UnitsNet/UnitDefinitions/ElectricResistivity.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "ElectricResistivity",
+  "BaseUnit": "OhmMeter",
+  "XmlDoc": "Electrical resistivity (also known as resistivity, specific electrical resistance, or volume resistivity) is a fundamental property that quantifies how strongly a given material opposes the flow of electric current.",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Electrical_resistivity_and_conductivity",
+  "Units": [
+    {
+      "SingularName": "OhmMeter",
+      "PluralName": "OhmsMeter",
```
[the remark this answers is not in the document]
Comment (baxcoders) [reply UnitsNet/UnitDefinitions/ElectricResistivity.json]: Fixed.

File: UnitsNet/UnitDefinitions/ElectricResistivity.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
```
[the remark this answers is not in the document]
Comment (baxcoders) [reply UnitsNet/UnitDefinitions/ElectricResistivity.json]: We added Nano, Micro and Milli as prefixes even if we didn't have any "real world" occurrence.

File: UnitsNet/UnitDefinitions/Illuminance.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "Illuminance",
+  "BaseUnit": "Lux",
+  "XmlDoc": "In photometry, illuminance is the total luminous flux incident on a surface, per unit area.",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Illuminance",
+  "Units": [
+    {
+      "SingularName": "Lux",
+      "PluralName": "Luxes",
```
[the remark this answers is not in the document]
Comment (baxcoders) [reply UnitsNet/UnitDefinitions/Illuminance.json]: Fixed.

File: UnitsNet/UnitDefinitions/Illuminance.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
```
[the remark this answers is not in the document]
Comment (baxcoders) [reply UnitsNet/UnitDefinitions/Illuminance.json]: We added Milli, Kilo and Mega as prefixes even if we didn't have any "real world" occurrence.

File: UnitsNet/UnitDefinitions/LuminousFlux.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  "BaseUnit": "Lumen",
+  "XmlDoc": "In photometry, luminous flux or luminous power is the measure of the perceived power of light.",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Luminous_flux",
+  "Units": [
+    {
+      "SingularName": "Lumen",
+      "PluralName": "Lumens",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
```
[the remark this answers is not in the document]
Comment (baxcoders) [reply UnitsNet/UnitDefinitions/LuminousFlux.json]: Never used anything different from "Lumens".

**Comment by angularsen** [comment]
Looks good, as I've commented earlier we generally prefer not to add single-unit quantities, but at least these quantities do have other units that can be added at a later stage.

**Pull request merged**